model-api-gen-runtime/src/commonMain/kotlin/org/modelix/metamodel/ITypedNode.kt

```diff
@@ -15,7 +15,7 @@ interface ITypedNode {
 fun ITypedNode.untyped() = unwrap()
 fun ITypedNode.untypedConcept() = _concept.untyped()
 fun ITypedNode.typedConcept() = _concept
-fun ITypedNode.getPropertyValue(property: IProperty): String? = unwrap().getPropertyValue(property.name)
```

Comment:
How does calling `getPropertyValue(property: IProperty)` instead of `getPropertyValue(role: String)` fixes the underlying bug?

Wouldn't the following example not still be broken?

```
val typedNode: ITypedNode = ...
typedNode.unwrap().getPropertyValue("someRole")
```

Reply:
`INode.getPropertyValue(String)` is deprecated and will not get fixed.